.env.example

```
@@ -13,4 +13,5 @@ DUMMY_USER_PASS=example123!
 
 # Optional
 TRUSTED_IPS=
+TRUSTED_CIDRS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRUSTED_CIDRS key should go before the TRUSTED_IPS key

(UnorderedKey)